src/main/resources/data-use-v3.json

```diff
@@ -0,0 +1,66 @@
+{
+  "id": "https://consent-ontology.dsde-prod.broadinstitute.org/schemas/data-use",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Use Schema V3",
+  "version": 2,
+  "type": "object",
+  "dependencies": {
+    "otherRestrictions": {
+      "anyOf": [
+        {"required": ["ethicsApprovalRequired"]},
+        {"required": ["collaboratorRequired"]},
+        {"required": ["geographicalRestrictions"]},
+        {"required": ["other"]}
+      ]
+    }
+  },
+  "properties": {
+    "generalUse": {
+      "type": "boolean"
+    },
+    "diseaseRestrictions": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "uniqueItems": true
+      }
+    },
+    "hmbResearch": {
+      "type": "boolean"
+    },
+    "populationOriginsAncestry": {
+      "type": "boolean"
+    },
+    "methodsResearch": {
+      "type": "boolean"
+    },
+    "commercialUse": {
+      "type": "boolean"
+    },
```

Comment:
I think I'd rather see an `NPU` case here. This has been a sticking point for a long time since we're not actually capturing non-profit use with this question. Alternatively, maybe we should capture `NPU` in addition to commercial. I can see these cases:
1. A dataset is available only for commercial use
2. A dataset is available only for non-profit use
3. A dataset is available for both non-profit and commercial use
Arguably, case 3 could be described with a `null` value in either case, but that is ambiguous.